core/commands/bitswap.go

```diff
@@ -0,0 +1,91 @@
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	cmds "github.com/jbenet/go-ipfs/commands"
+	bitswap "github.com/jbenet/go-ipfs/exchange/bitswap"
+	u "github.com/jbenet/go-ipfs/util"
+	"io"
+)
+
+var BitswapCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline:          "A set of commands to manipulate the bitswap agent",
+		ShortDescription: ``,
+	},
+	Subcommands: map[string]*cmds.Command{
+		"wantlist": showWantlistCmd,
+		"stat":     bitswapStatCmd,
+	},
+}
+
+var showWantlistCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Show blocks currently on the wantlist",
+		ShortDescription: `
+Print out all blocks currently on the bitswap wantlist for the local peer`,
+	},
+	Type: KeyList{},
+	Run: func(req cmds.Request, res cmds.Response) {
+		nd, err := req.Context().GetNode()
+		if err != nil {
+			res.SetError(err, cmds.ErrNormal)
+			return
+		}
+
+		res.SetOutput(&KeyList{nd.Exchange.GetWantlist()})
```

Comment:
this should still cast the exchange to bitswap, as this is a command called bitswap :)